assets/js/googlesitekit/modules/create-submit-changes-store.js

```diff
@@ -39,18 +36,14 @@ const FINISH_SUBMIT_CHANGES = 'FINISH_SUBMIT_CHANGES';
  * @since n.e.x.t
  *
  * @param {Object}   args                            Arguments for creating the submitChanges store.
- * @param {string}   args.storeName                  Datastore slug.
  * @param {Function} [args.submitChanges]            Optional. Callback function to issue the submit changes request. Will be used inside the submit changes control.
  * @param {Function} [args.validateCanSubmitChanges] Optional. A helper function to validate that settings can be submitted.
  * @return {Object} Partial store object with properties 'actions', 'controls', 'reducer', 'resolvers', and 'selectors'.
  */
 export function createSubmitChangesStore( {
-	storeName,
```

Comment:
Since we're removing this here, let's also make sure it isn't (unnecessarily) passed anymore from the callers.

Reply:
Good call! Updated 👍